packages/react-server/core/context/Navigator.js

```diff
@@ -83,45 +79,20 @@ class Navigator extends EventEmitter {
 
 			var loaders = route.config.page;
 
-			// Normalize.
-			// The page object may either directly be a loader or
-			// it may be an object whose values are loaders.
-			if (_.isFunction(loaders)){
-				loaders = {'default': loaders};
-			}
-
-			var mobileDetect = this.context.getMobileDetect();
-
 			// Our route may have multiple page implementations if
 			// there are device-specific variations.
 			//
 			// We'll take one of those if the request device
 			// matches, otherwise we'll use the default.
 			//
-			// Note that 'mobile' is the _union_ of 'phone' and
-			// 'tablet'.  If you _really_ want an iPad and an
-			// iPhone to get the _same_ non-desktop experience,
-			// use that.
-			//
-			var loadPage = [
-				'phone',
-				'tablet',
-				'mobile',
-			].reduce((loader, format) => {
-
-				// We'll take the _first_ format that matches.
-				if (!loader && loaders[format] && mobileDetect[format]()){
-
-					// Need to disambiguate for bundleNameUtil.
-					route.name += '-'+format;
```

Comment:
Do we not need this route.name override thing anymore for some reason?

Reply:
I guess maybe it's a product of eliminating `mobile`?

Reply:
Oh, nice catch. We use it for stat partitioning. I'll reintroduce that.